psiminer-core/src/main/kotlin/labelextractor/Label.kt

```diff
@@ -0,0 +1,17 @@
+package labelextractor
+
+interface Label{
+  fun getString(): String
```

Comment:
I suggest naming this method `getStringRepresentation` to show precisely the nature of the string.

Reply:
Thanks for accepting the solution!